src/components/LoanCards/BorrowerInfo/BorrowerInfoBody.vue

```diff
@@ -55,7 +55,7 @@ export default {
 		},
 		readMoreLinkText: {
 			type: String,
-			default: 'Read more',
+			default: 'Read full borrower details',
```

Comment:
If we don't want to change the default here you could also pass this as the `read-more-link-text` prop for this component in the desktop version of the expanded loan card